query-node/mappings/src/forum.ts

```diff
@@ -111,10 +112,20 @@ async function getPollAlternative(store: DatabaseManager, threadId: string, inde
 }
 
 async function getActorWorker(store: DatabaseManager, actor: PrivilegedActor): Promise<Worker> {
+  const workingGroup = await store.get(WorkingGroup, {
+    where: { id: 'forumWorkingGroup'},
+    relations: ['leader']
+  })
+
+  if (!workingGroup) {
+    throw new Error(`Forum Working Group not found!`)
+  }
+  
   const worker = await store.get(Worker, {
     where: {
       group: { id: 'forumWorkingGroup' },
       ...(actor.isLead ? { isLead: true } : { runtimeId: actor.asModerator.toNumber() }),
```

Comment:
I think there should be one `? :` operator,  not two.. 

Essentially, if `actor.isLead == true`, use the `leader.id` from the `workingGroup`, or else use moderator Id

Reply:
![image](https://user-images.githubusercontent.com/69617937/192262370-ba1aed99-b00d-4ca6-85ec-53c71339af46.png)
In https://github.com/Joystream/pioneer/issues/2832, I figured out two leaders in same group, so i tried to add new filter for it one more.

Reply:
Yes, I understand. But you don't need two filters. Only one is enough: 

`...(actor.isLead ? { id: workingGroup.leader?.id } : { runtimeId: actor.asModerator.toNumber() })`